pytng/pytng.pyx

```diff
@@ -597,9 +594,9 @@ cdef extern from "tng/tng_io.h":
         int64_t * n_values_per_frame,
         char * type) nogil
 
+# TODO REMOVE
```

Comment:
remove? is the frame tuple still used?

Reply:
I will remove the old API once we get this one in I think?

Reply:
Do you mean it should be removed as part of this PR or that once this PR is in you can work on other things to eventually remove it?

Reply:
Sorry should have been clearer.

I'll remove the old API in a separate PR once this PR is merged. There is a lot of stuff that can be cleaned up but we have to remove it all at once so it can all still compile.

Reply:
Thank you!